README.md

```diff
@@ -574,22 +574,28 @@ This repository contains a curated list of awesome open source libraries that wi
 * [InnerEye](https://innereye.ai/) - InnerEye combines human intelligence with artificial intelligence. By capitalizing on the merging of human neural processing and deep artificial neural networks, InnerEye allows fast and accurate visual inspection, real-time AI training and validation, and establishes a unique human-machine interface for connected user applications.
 * [Iguazio Data Science Platform](https://www.iguazio.com/) - Bring your Data Science to life by automating MLOps with end-to-end machine learning pipelines, transforming AI projects into real-world business outcomes, and supporting real-time performance at enterprise scale.
 * [Iterative Studio](https://studio.iterative.ai/) - Seamless data and model management, experiment tracking, visualization and automation, with Git as the single source of truth.
+* [Kaggle](https://www.kaggle.com) - Kaggle offers machine learning and data science competitions as well as offering a public data and cloud-based business platform for data science and AI education.
```

Comment:
![image](https://user-images.githubusercontent.com/8592144/223323983-7ec13a33-95de-444b-854f-c8f0e74cf1ed.png)

Reply:
I wouldn't really consider Kaggle as an MLOps vendor, even if extremely useful for the ML community - the way to think about this list would be to aim adding vendors that provide MLOps (ML productionisation capabilities)

Reply:
I still cannot understand well the concept of "productionisation". If a tool help with the production of machine learning deployment into real-world problems do we include it? If so, the Kaggle can be an MLOps tool to solve real-world problems. @axsaucedo

Reply:
I would see it as above, namely an "mlops vendor", at this stage it seems more of an "open" platform (ie owned by google so analogous to google colab) which is great but not a vendor that offers commercial mlops services

Reply:
It offer public dataset for ML deployment, as well as cloud computing for MLOps deployment, why shall we exclude it from "mlops vendor"? @axsaucedo

Reply:
I do think that if they indeed charged for their services they would be added to the list, but given it's free this is what raises my questions - I do think based on that I agree with you and it should be on the list